src/applications/mhv-landing-page/tests/e2e/confirm-email-address.cypress.spec.js

```diff
@@ -0,0 +1,149 @@
+import { rootUrl } from '../../manifest.json';
+import user from '../fixtures/user.json';
+import ApiInitializer from './utilities/ApiInitializer';
+
+const MHV_EMAIL_CONFIRMATION_DISMISSED_COOKIE =
+  'MHV_EMAIL_CONFIRMATION_DISMISSED';
+
+const clickConfirmEmail = () => {
```

Comment:
These functions should go in the page file `src/applications/mhv-landing-page/tests/e2e/pages/LandingPage.js`. This is pattern the VA wants. https://depo-platform-documentation.scrollhelp.site/developer-docs/best-practices-for-using-cypress#BestpracticesforusingCypress-PageObjects